### ISSUE TITLE
How to get full taxonomic classification like with kraken-report?

### ISSUE THREAD

**Issue by jamesriddellv** [bhattlab/kraken2_classification#27]
Hello,

I am interested in getting the domain-level classification for each sequence with kraken2.
With Kraken1 there was the kraken-translate option, but when trying to run that program with the kraken2 database, I get the error: 
`kraken-translate: database ("/fs/ess/*/*/Library/kraken-db7") does not contain necessary file database.kdb`
The kraken-db7 directory contains:
`hash.k2d  library  opts.k2d  seqid2taxid.map  taxo.k2d  taxonomy  unmapped.txt`

My current kraken2 output only gives the highest resolution taxonomic assignment. Is there a way to connect this to domain without needing database.kdb using only what is already available in my current install of the kraken2 database?

Thanks!

**Comment by bsiranosian** [comment]
Hi James, 

This may be a more appropriate question for the kraken2 developers, but I'll try my best. 

You can't use a kraken2 database (k2d file extensions) with the kraken1 programs. 

The standard kraken2 reports give percentages at multiple levels. For example the file *.krak.report will look like this, with percentages down the classification tree. If you use this pipeline, you'll also get classification matrices at different taxonomic levels. 

```
  2.52	161372	161372	U	0	unclassified
 97.23	6228989	3803	R	1	root
 97.16	6224466	2569	R1	131567	  cellular organisms
 97.06	6218282	422157	D	2	    Bacteria
 63.19	4048251	8	D1	1783270	      FCB group
 63.19	4048164	10	D2	68336	        Bacteroidetes/Chlorobi group
```

Is this what you're asking about?

**Comment by jamesriddellv** [comment]
Hi Ben,
Thank you for your fast reply. I was having trouble finding the kraken2 manual somehow and didn't realize this was separate. I found it now. Thanks for the info about the database differences. I needed the full taxonomic classification of each sequence ID for parsing fasta inputs and re-assembling by domain, which is lost in the aggregation by the krak report (only reports taxonomy without association to each sequence). I did some more digging and think the --use-mpa-style flag for kraken2 will give me all of the taxonomic levels for each sequence.
